app/serializers/user_serializer.rb

```diff
@@ -0,0 +1,15 @@
+class UserSerializer < ActiveModel::Serializer
+  attributes :id, :uid, :email, :username, :ideas, :contributions, :comments
+
+  def ideas
```

Comment:
you could try the has_many :ideas form for less verbosity, but it isn't a big deal